Naisten-Linja/naisten-linja-public-website
# Feature/add google tag manager

This PR resolves #53  as it adds Google Tag manager script to the site (as per requested in Slack).

This needs a new environment variable:
```bash
export GOOGLE_TAGMANAGER_ID=
```

and it needs to be added to Heroku as well, when we're deployin this. 

## How to test

You can check, that the script is indeed inserted by changing the `includeInDevelopment` to true from gatsby-config's plugin's resolver. Then, check the html-source of the page, and the script should be inside the `head`-tag.
